### PR TITLE
Proposal for renaming datacite rest API methods

### PR DESCRIPTION
**Pull request by elohanlon**
I thought it might be good to name the API methods for what they do rather than the REST verbs they use (like how in controllers we name the action methods "create" or "update" instead of "post" or "put").  Putting this out there as a change proposal.

### CODE REVIEW

File: docs/datacite.README.md
Comment (barmintor) [inline docs/datacite.README.md:103]: 😆  thank you for fixing my line numbers!

**Review by barmintor — APPROVED** [review]
This change makes sense to me. I'm tempted to suggest dropping the `_doi` suffixes, but the Datacite REST API does have some different resource types, so maybe worth holding off on that.

**Comment by barmintor** [comment]
I'm thinking a little more about this, and wondering if we might actually want to introduce new methods with these names that wrap the verbs, and hide the JSON response parsing from the client code. Does that make sense?

**Review by barmintor — COMMENTED** [review]
I'm thinking a little more about this, and wondering if we might actually want to introduce new methods with these names that wrap the verbs, and hide the JSON response parsing from the client code. Does that make sense?

**Review by fcd1 — APPROVED** [review]

**Comment by elohanlon** [comment]
Thank you both for the review!  As per our conversation, I've made some additional improvements!  I'll wait until the tests pass and then I'll merge the changes.

**Pull request merged**
